README.md

```diff
@@ -44,14 +44,16 @@ be declared so that the header is placed in the correct location in the image.
 Otherwise, the linker would place the header in a indeterminent location that
 would be unqiue for every build. An example is provided below.
 ~~~
+extern uintptr_t __start;
 img_hdr_t image_hdr __attribute__((section(".image_hdr"))) = {
     .image_magic = IMAGE_MAGIC,
     .image_hdr_version = IMAGE_VERSION_CURRENT,
     .image_type = IMAGE_TYPE_APP,
     .version_major = FW_VERSION_MAJOR,
     .version_minor = FW_VERSION_MINOR,
     .version_patch = FW_VERSION_PATCH,
-    .vector_addr = (uint32_t) &__start,
+    .vector_size = VECTOR_SIZE,
+    .vector_addr =  &__start,
     .device_id = IMAGE_DEVICE_ID_HOST,
```

Comment:
for documentation sake, lets move device_id above vector_size so it sits in the same spot as the definition.

Reply:
Sure. I will update. FYI, the order in declaration doesn't matter, but your comment is valid.